coralproject/cay
# Gallery preview save publish

## What does this PR do?
- Fixes https://github.com/coralproject/ask/issues/71
- Unifies gallery & form publish flow
## How do I test this PR?
- Go to a form with submissions and click on "Gallery"
- You'll see the publishing options on the right. 
- Try previewing, saving, and clicking on publish options for the embed codes.
- Try stopping Elkhorn to trigger an error when saving.

@coralproject/frontend
